tool/tctl/common/user_command.go

```diff
@@ -125,8 +130,8 @@ func (u *UserCommand) PrintSignupURL(client *auth.TunClient, token string) {
 		proxyPort = strconv.Itoa(defaults.HTTPListenPort)
 	}
 	url := web.CreateSignupLink(net.JoinHostPort(hostname, proxyPort), token)
-	fmt.Printf("Signup token has been created and is valid for %v seconds. Share this URL with the user:\n%v\n\nNOTE: make sure '%s' is accessible!\n",
-		defaults.MaxSignupTokenTTL.Seconds(), url, hostname)
+	fmt.Printf("Signup token has been created and is valid for %v hours. Share this URL with the user:\n%v\n\nNOTE: make sure '%s' is accessible!\n",
+		int(ttl/time.Hour), url, hostname)
```

Comment:
well... this could be more user friendly (it will probably print "0 hours" if you set ttl to 30 minutes) but lets roll